mods/ra/rules/structures.yaml

```diff
@@ -568,7 +568,7 @@ PBOX:
 		Armaments: garrisoned
 		PortOffsets: 384,0,128, 224,-341,128, -224,-341,128, -384,0,128, -224,341,128, 224,341,128
 		PortYaws: 0, 176, 341, 512, 682, 853
-		PortCones: 86, 86, 86, 86, 86, 86
+		PortCones: 100, 100, 100, 100, 100, 100
```

Comment:
As I said, 87 didn't work.Might try use lowest working. I have map with tons of unmoving engineers in every cell around pbox to test with.